pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -105,7 +105,7 @@ type ReconcileKnativeServing struct {
 // Create manifestival resources and KnativeServing, if necessary
 func (r *ReconcileKnativeServing) InjectClient(c client.Client) error {
 	koDataDir := os.Getenv("KO_DATA_PATH")
-	m, err := mf.NewManifest(filepath.Join(koDataDir, "knative-serving-0.6.0.yaml"), *recursive, c)
+	m, err := mf.NewManifest(filepath.Join(koDataDir, "knative-serving-0.7.0.yaml"), *recursive, c)
```

Comment:
I don't like having to update this file for every release. We can just agree on a directory that will always contain the correct version of knative-serving, and manifestival, like kubectl, will apply all the manifests it finds beneath that directory (typically just one):
```suggestion
	m, err := mf.NewManifest(filepath.Join(koDataDir, "knative-serving/"), false, c)
```